nncf/quantization/algorithms/weight_compression/algorithm.py

```diff
@@ -127,6 +131,34 @@ def _set_backend_entity(self, model: TModel) -> None:
                 "Cannot return backend-specific entity because {} is not supported!".format(model_backend.value)
             )
 
+    def _set_awq_backend_entity(
```

Comment:
I would recommend to move this code to `AWQAlgorithm`.

Reply:
Done